config/jobs/kubernetes/sig-testing/kubetest-canaries.yaml

```diff
@@ -218,13 +184,15 @@ periodics:
     preset-service-account: "true"
     preset-aws-ssh: "true"
     preset-aws-credential: "true"
+  decorate: true
+  decoration_config:
+    timeout: 14000000000000 # 140m
```

Comment:
I guess this one doesn't put everything because its not auto-generated?

Reply:
yes.

we basically load the entire prow config, and for each always_run k/k presubmit, generate a k/k-s presubmit spec, and dump them out.